src/otx/core/data/dataset/keypoint_detection.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (C) 2023 Intel Corporation
```

Comment:
```suggestion
# Copyright (C) 2024 Intel Corporation
```
It seems that manyfiles uses 2023. Could you update them also?

Reply:
fixed at 64b81f26eb328ddeeac0718f07d57d6c799cfab3

Reply:
Thanks but there are left thing still unchanged in other files. Could you check them also?

Reply:
5e3430c38c0baf01ad0f12721ff0afc0b777c3ce